posts/2020-09-10-wg-prio-call-for-contributors.md

```diff
@@ -0,0 +1,68 @@
+
+The day before the meeting the agenda is finalized and handed to the Compiler Team.
+
+Someone from the WG-Prioritization will then attend the meeting and provide details (if needed) and take down notes.
+
+Rinse and repeat for the next meeting.
+
+Everything is described in excruciatingly detail on [Rust Forge](https://forge.rust-lang.org/compiler/prioritization/procedure.html). Please have a look there to learn more.
+
+## How can I contribute?
+
+- **Help with issues prioritization**: keep an eye on the messages on our Zulip stream (~10 issues a week) and cast a vote on what the priority should be. Analyze the issue, figure out how the release could be impacted. More votes balance the prioritization. A couple of made-up examples:
+  - **ICE (Internal Compiler Error) on Rust stable channel for Tier3 supported platform**: unless it impacts Tier 1 platforms will likely be labelled as `P-low` (low priority). Will probably ping the relevant team and that's it until someone will pick it.
+  - **A "hello world" kind of code snippet fails to compile in the latest nightly**: that's a critical issue (`P-critical`) and we must stop this regression before it gets into the `beta` or (gosh!) `stable` channel. The relevant team will be swiftly alerted.
+  - **Obscure ICE when using some library**: if the issue doesn't smell bad, try assigning a `P-medium` and suggest the reporter to attach a `MCVE`.
+
+    With some experience, you will develop an _instinct_ to prioritize issues :-)
+
+- **Help properly summarize issues in the agenda**: what is this issue about? What has been already done to frame a context? Is is a regression? We add any detail that could be relevant to the Compiler team during their meeting. These folks are busy and could use all the help to get the context of an issue at a glance.
+
+## Ok, but can I actually contribute? I don't feel skilled enough
+
+Yes, you are! There will always be one or more members available to explain, mentor and clarify things. Don't be shy and do not refrain from asking questions.
+
+Everyone can contribute on their capacity and availability.
+
+## Where do we hang out
+
+One of the great things of the Rust governance is its openness. Join our stream [#t-compiler/wg-prioritization][zulip-wg-prio] and peek at how we work and if you want, also feel free to keep an eye to the weekly Team Compiler official meetings on [#t-compiler/meetings](https://rust-lang.zulipchat.com/#narrow/stream/238009-t-compiler.2Fmeetings). You can even simply just hang out on our Zulip stream, see how things work and then get involved when you feel to.
```

Comment:
Not sure “get involved when you feel to” is semantically correct, but I’m no native English so please dismiss the comment if it doesn’t make any sense ^^’

Reply:
Maybe
```suggestion
One of the great things of the Rust governance is its openness. Join our stream [#t-compiler/wg-prioritization][zulip-wg-prio] and peek at how we work and if you want, also feel free to keep an eye to the weekly Team Compiler official meetings on [#t-compiler/meetings](https://rust-lang.zulipchat.com/#narrow/stream/238009-t-compiler.2Fmeetings). You can even simply just hang out on our Zulip stream, see how things work and then get involved where you feel able.
```
or
```suggestion
One of the great things of the Rust governance is its openness. Join our stream [#t-compiler/wg-prioritization][zulip-wg-prio] and peek at how we work and if you want, also feel free to keep an eye to the weekly Team Compiler official meetings on [#t-compiler/meetings](https://rust-lang.zulipchat.com/#narrow/stream/238009-t-compiler.2Fmeetings). You can even simply just hang out on our Zulip stream, see how things work and then get involved when you have time.
```
depending on which you meant

Reply:
The first suggestion is probably closer to what I meant in my broken English: people can jump in when they feel they're ready.